docs/core/logger.md

```diff
@@ -519,6 +519,28 @@ We prioritise log level settings in this order:
 
 In the event you have set a log level in Powertools to a level that is lower than the ACL setting, we will output a warning log message informing you that your messages will be discarded by Lambda.
 
+### Setting timestamp to custom Timezone
+
+By default, this Logger and the standard logging library emit records with the default AWS Lambda timestamp in **UTC**.
```

Comment:
```suggestion
By default, Logger emits records with the default Lambda timestamp in **UTC**, i.e. `2016-06-20T12:08:10.000Z`
```

Reply:
Suggesting this change since we don't have a standard logging library in Node.js like Python does.